.github/workflows/e2e_tests.yml

```diff
@@ -194,7 +194,7 @@ jobs:
 
     strategy:
       matrix:
-        version: [1.18.14, 1.17.13, 1.16.15]
+        version: [1.18, 1.17]
```

Comment:
Sure 1.16 is not supported in AKS anymore?

Reply:
Yes according to this, https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar